src/soft-play.js

```diff
@@ -6,9 +6,59 @@ let children = 0
 // the adults and children variables defined above.
 // Start with the occupancy function.
 
+function occupancy() {
+  const total = { adults: adults, children: children }
+  return total
+}
+
+console.log(occupancy())
+
+const totalEntered = { adults: 0, children: 0 }
```

Comment:
this should be what `occupancy()` returns?

Reply:
The purpose of the totalEntered variable is to keep track of how many people are entering the softplay venue. This is my attempt at completing the soft-play extension exercise which is asking you to keep track of how many people entered. As the adults/children variable is updated every time someone goes in or out, this variable would not keep track of how many people entered. So I created a new variable for my function total() to return.